services/core-web/src/components/Forms/incidents/IncidentForm.js

```diff
@@ -910,6 +909,10 @@ const renderEditSaveControls = (childProps, isEditMode, isNewIncident) => (
 
 export const IncidentForm = (props) => {
   const [uploadedFiles, setUploadedFiles] = useState([]);
+  // 2nd parameter in getDropdownIncidentCategoryCodeOptions controls inclusion of historic categories
+  const incidentCategoryCodeOptions = useSelector((state) =>
+    getDropdownIncidentCategoryCodeOptions(state, false)
```

Comment:
Do we have a use-case where we do want to include the historic options?  Just wondering if the 2nd parameter is needed.

Reply:
I had to include them in order to display the legacy options!

This is what took me a ridiculous amount of time to wrap my head around- it already _does_ call for a second parameter, it just defaults to "true". That's done in createSelectorWrapper with the getVisibilityFilterOption, with the showActiveOnly parameter. I was waffling on the idea of including a link to something about selectors- on the one hand they are baffling but on the other hand they are a core functionality of Redux.

Reply:
Ah, okay, that makes sense to me!